triton_backend/inflight_batcher_llm/scripts/build.sh

```
@@ -52,7 +52,20 @@ if [[ "$BUILD_UNIT_TESTS" == "true" ]]; then
+TRITON_SHORT_TAG="main"
+if [[ -f "$DOCKERFILE_MULTI_PATH" ]]; then
+  TRITON_BASE_TAG_LINE=$(grep -E '^ARG TRITON_BASE_TAG=' "$DOCKERFILE_MULTI_PATH" | tail -n1)
+  if [[ -n "$TRITON_BASE_TAG_LINE" ]]; then
+    TRITON_BASE_TAG=$(echo "$TRITON_BASE_TAG_LINE" | cut -d'=' -f2)
+    if [[ -n "$TRITON_BASE_TAG" ]]; then
+      TRITON_SHORT_TAG="r${TRITON_BASE_TAG%-py}"
+      echo "Using triton tag from Dockerfile.multi: $TRITON_SHORT_TAG"
+    fi
+  fi
+fi
+cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install ${BUILD_TESTS_ARG} -DTRITON_COMMON_REPO_TAG=${TRITON_SHORT_TAG} -DTRITON_CORE_REPO_TAG=${TRITON_SHORT_TAG} -DTRITON_THIRD_PARTY_REPO_TAG=${TRITON_SHORT_TAG} -DTRITON_BACKEND_REPO_TAG=${TRITON_SHORT_TAG} ..
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)